Userland/Libraries/LibWeb/HTML/Navigable.h

```diff
@@ -42,6 +42,8 @@ class Navigable : public JS::Cell {
     JS::GCPtr<WindowProxy> active_window_proxy();
     JS::GCPtr<Window> active_window();
 
+    JS::GCPtr<SessionHistoryEntry> get_the_target_history_entry(int) const;
```

Comment:
Please specify the parameter name here. `target_step` is not inferrable from context :)